json-templates/vpc-access.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
I just added this file to have an example for VPC policy. Not necessary for this feature.